fastlane/README.md

```diff
@@ -184,12 +184,6 @@ See how [Wikipedia](https://github.com/fastlane/examples#wikipedia-by-wikimedia-
 - [`gym`](https://github.com/fastlane/fastlane/tree/master/gym): Building your iOS apps has never been easier
 - [`match`](https://github.com/fastlane/fastlane/tree/master/match): Easily sync your certificates and profiles across your team using git
 
-## Statistics
```

Comment:
Do you think we should keep the header here, and just link to the main readme's section? Do we link to this page `#statistics` anywhere? Do we think anybody links to this? 

Not saying that we should, just curios on what everybody thinks.